homeassistant/components/device_tracker/netgear.py

```diff
@@ -12,21 +12,29 @@
 from homeassistant.components.device_tracker import (
     DOMAIN, PLATFORM_SCHEMA, DeviceScanner)
 from homeassistant.const import (
-    CONF_HOST, CONF_PASSWORD, CONF_USERNAME, CONF_PORT)
+    CONF_HOST, CONF_PASSWORD, CONF_USERNAME, CONF_PORT, CONF_URL,
+    CONF_DEVICES, CONF_EXCLUDE)
 
-REQUIREMENTS = ['pynetgear==0.3.3']
+REQUIREMENTS = ['pynetgear==0.3.4']
 
 _LOGGER = logging.getLogger(__name__)
 
-DEFAULT_HOST = 'routerlogin.net'
+CONF_APS = 'accesspoints'
+
 DEFAULT_USER = 'admin'
```

Comment:
If you move everything to pynetgear then this should be move too.

Reply:
You mean let pynetgear handled the default for the username ?